controllers/categories.controllers.js

```diff
@@ -0,0 +1,9 @@
+const { selectCategories } = require('../models/categories.models');
+
+exports.getCategories = (req, res, next) => {
+    selectCategories()
+    .then((categories) => {
+        res.send(categories);
```

Comment:
Almost there, we just need to send the data back in an object, with a key of categories

Reply:
Also need a status 200